dockerspawner/dockerspawner.py

```diff
@@ -147,6 +147,18 @@ def client(self):
         )
     )
 
+    internal_ip_network_name = Unicode(
+        "bridge",
+        config=True,
+        help=dedent(
+            """
+            Retrieve the internal IP from this docker network. Defaults to the default docker
```

Comment:
Should this be `network` or `network_name`? Also, the first line should probably start with:

> The name of the docker network from which to retrieve the internal IP address.

From this first line and the variable name, I expected this trait to be an IP address, not the name of the network.